pkg/controller/registry/resolver/steps.go

```diff
@@ -136,6 +154,12 @@ func NewStepResourceFromBundle(bundle *api.Bundle, namespace, replaces, catalogS
 	}
 	steps := []v1alpha1.StepResource{step}
 
+	step, err = NewOperatorConditionStepResource(csv, catalogSourceName, catalogSourceName)
```

Comment:
What happens if/when an operator author includes an `OperatorCondition` in their bundle?

Reply:
I'm glad you brought that up. Since each `OperatorCondition` is created for a `CSV` in the installplan I believe that it makes sense to update the `isSupported` feature to make sure that the operatorCondition name matches a CSV included in the installplan.

Reply:
No longer relevant.